go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/consensys/bavard v0.1.31-0.20250406004941-2db259e4b582
 	github.com/consensys/compress v0.2.5
-	github.com/consensys/gnark-crypto v0.17.1-0.20250502112255-56600883e0e9
+	github.com/consensys/gnark-crypto v0.17.1-0.20250515203007-25d707ef7020
```

Comment:
This commit is out of gnark-crypto master, i.e. do not merge before the dependency on gnark-crypto side is merged!